.github/workflows/validate.yaml

```diff
@@ -7,7 +7,7 @@ jobs:
     runs-on: windows-latest
     name: Check Formatting
     steps:
-      - uses: actions/checkout@v3
```

Comment:
actions/checkout@v3 is deprecated and will be removed as it uses node 16, v4 uses node 20 so probably valid for at least a coupe of years...